ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/StatementExecutor.java

```diff
@@ -193,10 +193,10 @@ private void executeStatement(
       final PersistentQueryMetadata query = startQuery(statement, command, mode);
       successMessage = statement.getStatement() instanceof CreateTableAsSelect
           ? "Table created and running" : "Stream created and running";
-      successMessage += " with id: " + query.getQueryId();
+      successMessage += " with query ID: " + query.getQueryId();
```

Comment:
Maybe change the message from `Stream created and running with queryID: blaa` to something like `Stream created by the query with queryID: blaa` since the query is running not the stream or table!